src/Adapter/Driver/Pdo/Pdo.php

```diff
@@ -37,6 +37,9 @@ class Pdo implements DriverInterface, DriverFeatureInterface, Profiler\ProfilerA
     /** @var array */
     protected $features = [];
 
+    /** @var Profiler\ProfilerInterface */
+    protected $profiler;
```

Comment:
need to have `public` modifier, unfortunatelly, ref https://3v4l.org/HYEML

Reply:
There are public getProfiler() method, but sure, lets not introduce BC break.